docs/formatter.md

```diff
@@ -23,7 +21,7 @@ and instead exit with a non-zero status code upon detecting any unformatted file
 For the full list of supported options, run `ruff format --help`.
 
 !!! note
```

Comment:
Perhaps this shouldn't be a note any more, should it?

Reply:
Yeah, I think we can remove the entire note here